entrypoint/go.mod

```diff
@@ -1,10 +1,33 @@
 module entrypoint
 
-go 1.14
+go 1.22
 
 require (
 	github.com/Icinga/go-libs v0.0.0-20220420130327-ef58ad52edd8
-	github.com/google/go-cmp v0.5.6 // indirect
-	github.com/icinga/icingadb v1.0.0
-	go.uber.org/zap v1.22.0
+	github.com/icinga/icingadb v1.2.0
+	go.uber.org/zap v1.27.0
+)
+
+require (
+	filippo.io/edwards25519 v1.1.0 // indirect
+	github.com/cespare/xxhash/v2 v2.2.0 // indirect
+	github.com/creasty/defaults v1.7.0 // indirect
+	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
+	github.com/fatih/color v1.13.0 // indirect
+	github.com/go-sql-driver/mysql v1.8.1 // indirect
+	github.com/goccy/go-yaml v1.11.3 // indirect
+	github.com/google/uuid v1.6.0 // indirect
+	github.com/jessevdk/go-flags v1.5.0 // indirect
+	github.com/jmoiron/sqlx v1.3.5 // indirect
+	github.com/lib/pq v1.10.9 // indirect
+	github.com/mattn/go-colorable v0.1.13 // indirect
+	github.com/mattn/go-isatty v0.0.16 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/redis/go-redis/v9 v9.5.1 // indirect
+	github.com/ssgreg/journald v1.0.0 // indirect
+	go.uber.org/multierr v1.10.0 // indirect
+	golang.org/x/exp v0.0.0-20220613132600-b0d781184e0d // indirect
+	golang.org/x/sync v0.7.0 // indirect
+	golang.org/x/sys v0.6.0 // indirect
+	golang.org/x/xerrors v0.0.0-20220907171357-04be3eba64a2 // indirect
```

Comment:
What's this for?

Reply:
I think that's because I merged changes from the main branch back to my feature branch.

Reply:
Please undo.

Reply:
I force pushed my last good commit. 
So you shouldn't see the golang related diffs any more.